### PR TITLE
Third party notices

### PR DESCRIPTION
**Pull request by nijotz**
* Added third party notices file.

* Updated metric retrieval from agent for `assert.metrics` to match newer agent versions.

### CODE REVIEW

**Comment by nijotz** [comment]
I also updated the license field in package.json in this PR

**Comment by nijotz** [comment]
Added footer to notices file

**Comment by nijotz** [comment]
hmm, something about this may hang the travis tests?

File: THIRD_PARTY_NOTICES.md [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+license, which is reproduced above.
+
+Copyright (c) 2016 Yoshihide Jimbo.
+
+The source code for the Tomorrow theme is available at:
+https://github.com/jmblog/color-themes-for-google-code-prettify
+
+```
+
+### newrelic
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES.md:434]: Thinking we should bump the newrelic dev dependency on this project, as well, so have matching Apache 2.0 licenses.

File: THIRD_PARTY_NOTICES.md [hunk 2]
```diff
@@ -0,0 +1,811 @@
...
+ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF OR
+IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+```
+
+-----
+
+Portions copyright Node.js contributors. Depending on your existing libraries and package management settings,
```
Comment (michaelgoin) [inline THIRD_PARTY_NOTICES.md:808]: Thanks for grabbing this 👍

File: THIRD_PARTY_NOTICES.md (hunk 1 above)
Comment (nijotz) [reply THIRD_PARTY_NOTICES.md:434]: Bumped to open source version

**Comment by nijotz** [comment]
> Not sure what you did to anger Travis. May need to land actions PR and merge into this haha.

haha, ya maybe the package.json changing in some way? Let me check tests locally

**Comment by nijotz** [comment]
Had to update tests to handle new metrics interface

File: lib/assert.js [hunk 3]
```diff
@@ -223,11 +223,12 @@ function _metricsTest(expected, exact) {
     }
   }
 
-  if (exact && metrics.toJSON().length !== expected.length) {
+  let metricsJSON = metrics._metrics.toJSON()
```
Comment (michaelgoin) [inline lib/assert.js:226]: interesting... i'm surprised this was working for tests? perhaps this code path isn't used by any external modules currently. i assumed the failure would be in the code testing this module itself not in the actual assert helpers for external tests.

**Pull request merged**